PayPalWebPayments/src/main/java/com/paypal/android/paypalwebpayments/PayPalWebCheckoutClient.kt

```diff
@@ -13,7 +14,7 @@ import com.paypal.android.corepayments.analytics.AnalyticsService
  */
 class PayPalWebCheckoutClient internal constructor(
     // NEXT MAJOR VERSION: remove hardcoded activity reference
-    private val activity: FragmentActivity,
+    activity: FragmentActivity,
```

Comment:
Does this need to be public now?

Reply:
it is no longer a class variable - it's a linter optimization

Reply:
Appears it can remain a private property. Pushing changes now.